template/src/ontology/Makefile.jinja2

```diff
@@ -26,8 +26,11 @@ EDIT_FORMAT=                {{ project.edit_format|default('owl') }}
 SRC =                       $(ONT)-edit.$(EDIT_FORMAT)
 ROBOT=                      robot
 RELEASEDIR=                 ../..
-PATTERNDIR=                 ../patterns
-REPORTDIR=                 reports
+{% if project.pattern_pipelines_group %}
+PATTERNDIR=                 {{ project.pattern_pipelines_group.pattern_directory }}
```

Comment:
I dont think this will work; at least not in its current form. The current pipeline assumes 1 pattern directory, which is in PATTERNDIR/dosdp-patterns, and an arbitrary number of pipeline directories, which can be found under patterndir/data/. There is currently no support for having a seperate pattern directory for every single pipeline. Is that every necessary? Do you want to be able to use two different patterns with the same name?

Reply:
So as an example, we currently have
- ontology
- patterns
  - dosdp-patterns [abnormal.yaml, b.yaml, c.yaml]
  - data
    - pipeline 1 [abnormal.tsv]
    - pipeline 2 [b.tsv, c.tsv]
    - pipeline 3 [b.tsv]

Reply:
This is to support ontologies that already place the patterns dir somewhere else, or have patterns and tsvs in separate folders.E.g. envo.  If it's too complex they could simply git mv

Reply:
Patterns and tsvs are in different folders right now! So it is only about the exact location of the patterns directory? I kind of like this to be standard, no? or do you think it is an effort for people to move their patterns to this patterns/dosdp-patterns directory?